metals/src/main/scala/scala/meta/metals/Workspace.scala

```diff
@@ -41,5 +41,8 @@ object Workspace {
         }
       }
     )
+    Option(sbtserver.SbtServer.activeJson(cwd))
+      .filter { active => Files.exists(active.toNIO) }
```

Comment:
`(_.isFile)`

Reply:
I will include it in the `SbtServer.ActiveJson()` unapply